### PR TITLE
Feature/2056 sewio clients

### PR DESCRIPTION
**Pull request by davidnich**

### CODE REVIEW

File: qlib/SewioWebSocketClient.qm [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+            - \c debuglog: (optional) @ref closure "closure" or @ref call_reference "call reference" for logging debugging/technical messages about the web socket connection
+
+            @throw WEBSOCKET-ERROR unknown scheme, missing 'url' key in option hash
+        */
+        constructor(code cb, hash opts) : WebSocketClient(SewioWebSocketClient::getCallback(cb), opts) {
+            # check required options
+            foreach string key in (RequiredOptions) {
+                any v = opts{key};
+                if (!exists v || v == "")
+                    throw "SEWIOWEBSOCKETCLIENT-ERROR", sprintf("missing required option %y in option argument", key);
```
Comment (omusil24) [inline qlib/SewioWebSocketClient.qm:170]: Missing in docs

File: qlib/SewioWebSocketClient.qm [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+
+    Currently the module provides the following classes:
+    - @ref SewioWebSocketClient::SewioWebSocketClient "SewioWebSocketClient": the Sewio RTLS Studio web socket client class
+    - @ref SewioWebSocketClient::SewioWebSocketConnection "SewioWebSocketConnection": the Sewio RTLS Studio web socket connection class (based on the @ref connectionproviderintro "ConnectionProvider" module)
+
+    <b>Example</b>
+    @code{.py}
+#!/usr/bin/env qore
+
+%require-our
```
Comment (omusil24) [inline qlib/SewioWebSocketClient.qm]: Really this one?

**Comment by davidnich** [comment]
@omusil24 fixed

File: qlib/SewioWebSocketClient.qm (hunk 1 above)
Comment (davidnich) [reply qlib/SewioWebSocketClient.qm:170]: resolved

File: qlib/SewioWebSocketClient.qm (hunk 2 above)
Comment (davidnich) [reply qlib/SewioWebSocketClient.qm]: resolved

**Pull request merged**